packages/wallets/src/features/cashier/modules/Transfer/components/TransferFormAccountCard/TransferFormAccountCard.tsx

```diff
@@ -53,8 +52,8 @@ const TransferFormAccountCard: React.FC<TProps> = ({ account, activeWallet, type
                         />
                     )}
                 </div>
-                {isInput && isMobile && (
-                    <WalletListCardBadge isDemo={Boolean(account?.demo_account)} label={badgeLabel} />
+                {isInput && isMobile && !!account?.demo_account && (
+                    <WalletListCardBadge isDemo={Boolean(account?.demo_account)} label='virtual' />
```

Comment:
What happens if `isDemo={true}` is the label still be virtual?

Reply:
@aizad-deriv, Nope, the text on the badge is displayed as `Demo`. Check [this](https://github.com/binary-com/deriv-app/pull/15836/files#:~:text=const%20formattedLabel%20%3D%20label%20%3D%3D%3D%20%27virtual%27%20%3F%20%27Demo%27%20%3A%20label%3F.toUpperCase()%3B) line for reference.